src/v/features/feature_table.cc

```diff
@@ -232,10 +232,10 @@ void feature_state::notify_version(cluster_version v) {
     }
 }
 
-void feature_table::set_active_version(cluster_version v) {
+void feature_table::set_active_version(cluster_version v, bool ephemeral) {
```

Comment:
nit: maybe consider some enum like `version_setting_type{ cluster_driven, local_only }` or somesuch?

Reply:
Good idea, I've switched this  bool to an enum.